app/assets/stylesheets/home.scss

```diff
@@ -40,113 +57,111 @@ h3 {
   background-color: #00796B;
 }
 
-#team {
-  background: image_url("success_team.png");
-  background-size: cover;
-  height: 150px;
-  width: 150px;
+li.team {
+  text-align: center;
+  padding-right: 0;
+  padding-left: 25px;
+  width: 250px;
 }
 
-.site-footer {
-  background-color: #00796B;
-  color: white;
+#team {
+  padding-right: 0;
 }
 
-* {
-	margin: 0;
-}
-html, body {
-	height: 100%;
+li.search {
+  text-align: center;
```

Comment:
Properties should be ordered float, margin-right, margin-top, max-width, padding, text-align